### PR TITLE
Allow starting servers via app's config

### PR DESCRIPTION
**Pull request by arkgil**
Example:

```elixir
config :fennec, :servers,
  udp: [port: 32123, ip: {127, 0, 0, 1}]
```

Addresses #4

### CODE REVIEW

**Comment by arkgil** [comment]
@mentels  could you review that one?

File: lib/fennec/application.ex [hunk 1]
```diff
@@ -3,12 +3,26 @@ defmodule Fennec.Application do
 
   use Application
 
+  import Supervisor.Spec
+
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
```
Comment (mentels) [inline lib/fennec/application.ex]: Two empty lines?

File: lib/fennec/application.ex [hunk 2]
```diff
@@ -3,12 +3,26 @@ defmodule Fennec.Application do
...
 
-    children = []
 
     opts = [strategy: :one_for_one, name: Fennec.Supervisor]
-    Supervisor.start_link(children, opts)
+    {:ok, pid} = Supervisor.start_link(servers(), opts)
```
Comment (mentels) [inline lib/fennec/application.ex]: Why returning that? ERTS will take care of the system if this call fails.

File: lib/fennec/application.ex [hunk 3]
```diff
@@ -3,12 +3,26 @@ defmodule Fennec.Application do
...
+  end
+
+  defp servers do
+    :fennec
+    |> Application.get_env(:servers, [])
```
Comment (mentels) [inline lib/fennec/application.ex]: Piping an atom doesn't feel like data transformation. But it's my personal feeling. Don't treat as a MUST

File: lib/fennec/udp.ex [hunk 4]
```diff
@@ -2,8 +2,13 @@ defmodule Fennec.UDP do
   @moduledoc """
   UDP STUN server
 
-  The easiest way to start a server is to hook it up
-  to your supervision tree:
+  The easiest way to start a server is to start it under Fennec's
+  supervision tree using configuration:
```
Comment (mentels) [inline lib/fennec/udp.ex]: ...is to spawn it under Fennec's supervision tree providing the following configuration...

File: lib/fennec/udp.ex [hunk 5]
```diff
@@ -14,6 +19,13 @@ defmodule Fennec.UDP do
   You can also start a server under Fennec's supervision tree
   using `start/1`.
 
+  ## Options
+
+  All methods of starting a server accepts the same configuration options
```
Comment (mentels) [inline lib/fennec/udp.ex]: All the methods...accept..passed as a keyword list:
...
... of an interface ... listen on

**Pull request merged**